### PR TITLE
Assessment edits

### PR DESCRIPTION
**Pull request by rtillies**

### CODE REVIEW

File: SQL.txt [hunk 1]
```diff
@@ -63,17 +63,17 @@ VALUES
...
-3. List all songs that have been in the number 1 position (peakPosition)
+3. List all songs that have been in the number 1 position (peakPosition).
 < YOUR QUERY >
 
-4. Remove All songs by `the Chainsmokers`
+4. Remove All songs by `The Chainsmokers`.
 < YOUR QUERY >
 
-5. Update Post Malone's song `Congratualtions` to have a peakPosition of 3.
+5. Update Post Malone's song `Congratulations` to have a peak position of 3.
```
Comment (rtillies) [inline SQL.txt]: I go back and forth with this. Students have to use the _slightest_ bit of critical thinking to connect "peak position" to the `peakPosition` field, but it should be fairly obvious.

File: SQL.txt [hunk 2]
```diff
@@ -63,17 +63,17 @@ VALUES
 	(28,'XXXTENTACION','Moonlight',923,2);
```
Comment (rtillies) [inline SQL.txt:64]: Line 62: How do you feel about the profanity? Should we censor the song title in this case?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Mod2 Week1 Assessment - <YOUR NAME HERE>
 
 ## Setup
-1. Fork this repository
-2. Open your forked repository in Visual Studio
+1. Fork this repository.
+2. Open your forked repository in Visual Studio.
 
 ## Instructions
 1. Insert your name on line 1 to replace `<YOUR NAME HERE>` above.
```
Comment (rtillies) [inline README.md]: It seems obvious to us, but students often make the mistake of not removing the angle brackets. Thus it becomes
`Mod2 Week1 Assessment - <Megan McMahon>`. Do we need to be more explicit about this (not just here, across the board)?

Comment (memcmahon) [reply README.md]: Yea, I don't think it hurts to be more specific.

File: SQL.txt (hunk 2 above)
Comment (memcmahon) [reply SQL.txt:64]: Hmmm.... this is a good question..... My personal feeling is that our students will be over 18, so I'm less inclined to censor.  I can also see how this is maybe a bit cringy for a published lesson.  

And, I am always down for an easter egg... so I'll fracking change it :)

File: SQL.txt (hunk 1 above)
Comment (memcmahon) [reply SQL.txt]: I think they should be able to make this leap :)

**Review by memcmahon — APPROVED** [review]
Thanks for the review!

**Pull request merged**